tests/phpunit/tests/functions/_wpCheckExistingFileNames.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * @group functions.php
+ * @covers ::_wp_check_existing_file_names
```

Comment:
Per [56070](https://core.trac.wordpress.org/ticket/56070), There should be a line break between them.
```suggestion
 * @group functions.php
 *
 * @covers ::_wp_check_existing_file_names
```